autogluon/utils/tabular/utils/loaders/load_pd.py

```diff
@@ -35,6 +36,12 @@ def load(path, delimiter=',', encoding='utf-8', columns_to_keep=None, dtype=None
         format = 'parquet'
     else:
         format = 'csv'
+        if path.endswith('.tsv'):
+            if delimiter is None:
+                delimiter = '\t'
```

Comment:
Should we add logger debug message stating the delimiter was inferred (at L41 here and L44)?  I imagine the official error message may be opaque if the delimiter is wrong.

Reply:
Good point, added